generators/src/new-test-model/templates/model/builders.spec.tpl

```diff
@@ -0,0 +1,27 @@
+import { {{modelName}}Rest, {{modelName}}Graphql } from './index';
```

Comment:
Just from a high-level, not that much speaks against using Mustache. Do we need it? We basically do string interpolation without any logic like helpers, partials or logic (like the golden days).

As we use Lodash already and lodash comes with a template function which would maybe be enough already [ref](https://lodash.com/docs/4.17.15#template). Would that also work to avoid using another library?

Reply:
I'm all for not adding new libraries if not needed so let me take a look at lodash capabilities.

Perhaps this is a topic to seek for an agreement as I already found another templating library ([Squirrely](https://github.com/squirrellyjs/squirrelly)) used in the presets generation repository.

Reply:
Squirrelly also looks nice in lean in terms of offering the minimal set of thing we need. Lodash also allows this inline JavaScript, looping and conditionals. If I'd had to choose I must admit Squirrelly looks more focussed a single task which I like.

Reply:
Ok, the I'll refactor the CLI to use Squirrely

Reply:
Migrated from Mustache to Squirrelly here: a2ff62e5806558737268dc4b6f868a933f0f92a8